.github/workflows/docs.yml

```diff
@@ -46,11 +46,6 @@ jobs:
 
       - name: Configure git
         run: |
-          # Get the master branch so we can run benchmarks on it
```

Comment:
Moved this so that we check out to `master` only when generating the benchmarks. Otherwise there was an error popping up, see https://github.com/django-components/django-components/actions/runs/13942214025/job/39021265205